client/analytics/settings/config.js

```diff
@@ -0,0 +1,62 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { applyFilters } from '@wordpress/hooks';
+
+const SETTINGS_FILTER = 'woocommerce_admin_analytics_settings';
+
+const defaultOrderStatuses = [
+	'completed',
+	'processing',
+	'refunded',
+	'cancelled',
+	'failed',
+	'pending',
+	'on-hold',
+];
+const orderStatuses = Object.keys( wcSettings.orderStatuses )
+	.filter( status => status !== 'refunded' )
+	.map( key => {
+		return {
+			value: key,
+			label: wcSettings.orderStatuses[ key ],
+			description: sprintf(
+				__( 'Exclude the %s status from reports', 'wc-admin' ),
+				wcSettings.orderStatuses[ key ]
+			),
+		};
+	} );
+
+export const analyticsSettings = applyFilters( SETTINGS_FILTER, [
+	{
+		name: 'woocommerce_excluded_report_order_statuses',
+		label: __( 'Excluded Statuses:', 'wc-admin' ),
+		inputType: 'checkboxGroup',
+		options: [
+			{
+				key: 'defaultStatuses',
+				options: orderStatuses.filter(
+					status => defaultOrderStatuses.includes( status.value ) && 'refunded' !== status.value
```

Comment:
Nitpick: `refunded` status is already filtered out above.

Reply:
Good catch!  Will get this change made.